docs/api/Layer.md

```diff
@@ -34,6 +34,18 @@ A new identical layer will be inserted into the parent of this layer.
 
 A new [Layer](#layer).
 
+## Convert the Layer to Outlines
+
+```js
+const layers = layer.layersByConvertingToOutlines()
+```
+
+Convert this layer to outlines.
+
+### Returns
+
+An array of new [Layer](#layer) objects representing the outlines of this layer.
```

Comment:
Might be nice to clarify when this has more than one layer. I'm still not sure when that would be.